src/styles/index.scss

```
@@ -1,3 +1,128 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: block;
+  width: 198px;
+  height: 406px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  justify-content: center;
+  align-items: center;
+}
+
+.monitor {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 36px;
+}
+
+.description {
+  justify-content: center;
+  align-items: center;
+  padding: 0 16px;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.code {
+  padding: 4px 84px 0 16px;
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  word-wrap: break-word;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding: 16px 16px 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.rewiev {
+  color: #060b35;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  word-wrap: break-word;
+}
+
+.mark {
+  padding: 24px 16px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+}
+
+.price {
+  color: #616070;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  word-wrap: break-word;
+}
+
+.sum {
+  color: #060b35;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  word-wrap: break-word;
+}
+
+.button {
+  margin: 16px;
+}
+
+.buy {
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  line-height: 16px;
+  word-wrap: break-word;
+}
+
+.buy:hover {
+  background-color: #f3f3f3;
+  color: #00acdc;
+}
```

No need to write duplicates classes
```suggestion
.buy {
  height: 40px;
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: #00acdc;
  border-radius: 5px;
  border: 1px solid #00acdc;
  color: white;
  font-size: 14px;
  font-weight: 700;
  text-transform: uppercase;
  line-height: 16px;
  word-wrap: break-word;
  
  &:hover {
    background-color: #f3f3f3;
    color: #00acdc; 
  }
}
```